src/data/nat/basic.lean

```diff
@@ -20,6 +20,11 @@ and extra recursors:
 
 universes u v
 
+run_cmd do env ← tactic.get_env,
+  tactic.set_env $
+  [``nat.pow_zero,
+   ``nat.pow_succ].foldl environment.mk_protected env
+
```

Comment:
How about just fixing these directly in core?

Reply:
We could do that easily in a follow-up PR.